docs/en/ingest-management/processors/processor-communityid.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[community_id-processor]]
+= Community ID Network Flow Hash
+
+++++
+<titleabbrev>community_id</titleabbrev>
+++++
+
+The `community_id` processor computes a network flow hash according to the
+https://github.com/corelight/community-id-spec[Community ID Flow Hash
+specification].
+
+The flow hash is useful for correlating all network events related to a
+single flow. For example, you can filter on a community ID value and you might
+get back the Netflow records from multiple collectors and layer 7 protocol
+records from Packetbeat.
+
+//REVIEWERS: Is it OK to mention Packetbeat here?
```

Comment:
Packetbeat still exists outside of Agent so that statement seems valid. If we want to rewrite it with only Agent is mind then we could replace "from Packetbeat" with  "from the Network Packet Capture integration".